### PR TITLE
Adding PS Cmdlets for Azure Automation Python3 operation

### PR DESCRIPTION
**Pull request by sushil490023**
## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by wyunchi-ms** [comment]
1. Please update Changelog,md
2. [krmanupa](https://github.com/krmanupa) just upgrade the SDK in this sprint. Please confirm the API version with him.

File: src/Automation/Automation.Sdk/README.md [hunk 1]
```diff
@@ -25,31 +25,32 @@ title: AutomationClient
...
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/connectionType.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/sourceControl.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/sourceControlSyncJob.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/sourceControlSyncJobStreams.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/dscConfiguration.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/updatePython3Swagger/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/job.json
```
Comment (wyunchi-ms) [inline src/Automation/Automation.Sdk/README.md:44]: Please don't use the swagger in your own repo.

Comment (sushil490023) [reply src/Automation/Automation.Sdk/README.md]: Sorry i have updated the reference from azure repo. Thanks for pointing out.

**Comment by wyunchi-ms** [comment]
1. Missed help file for new cmdlets.
2. Please re-record the test cases.

**Comment by sushil490023** [comment]
> 

1. Added help files
2. Update session record files for test cases

**Comment by wyunchi-ms** [comment]
Hi @sushil490023 , I see there is still issues in the static analysis for the new cmdlets. Modifying the resources will need support `ShouldProcess` in PowerShell module.
The test cases still fails. Do you follow the steps in our [guideline](https://github.com/Azure/azure-powershell/blob/main/documentation/testing-docs/using-azure-test-framework.md#manually-set-environment-variables)?

**Comment by VeryEarly** [comment]
please also sync your branch with latest main

**Comment by sushil490023** [comment]
@microsoft-github-policy-service agree [company="microsoft"]

**Review by VeryEarly — DISMISSED** [review]

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**